README.md

```diff
@@ -75,6 +75,11 @@ In config file, change `useIdeasFromSettings: false,` to `useIdeasFromSettings:
 #### Editing break/microbreak notification interval
 In config file, change `breakNotificationInterval: 30000,` to whatever value you want. 30000 is 30 seconds. Same goes for microbreak.
 
+#### Editing sunrise time to pause breaks until morning
+In config file you can set morningHour setting to pause until that hour this or next day
```

Comment:
let's have code formatting here with "`" for variables like  `morningHour` and so. 

maybe add a little example? ie:

If you live in Boston, you would set 
morningHour: "sunrise", 
posLatitude: xxx, 
posLongitude: yyy